seatunnel-connectors-v2/connector-pulsar/src/main/java/org/apache/seatunnel/connectors/seatunnel/pulsar/source/PulsarSource.java

```diff
@@ -222,7 +225,9 @@ private void setPartitionDiscoverer(Config config) {
 
     private void setDeserialization(Config config) {
         String format = config.getString("format");
-        // TODO: json format
+        // TODO: format SPI
+        SeaTunnelRowType rowType = SeatunnelSchema.buildWithConfig(config.getConfig(SeatunnelSchema.SCHEMA)).getSeaTunnelRowType();
```

Comment:
Use `fields` as config key?  reference #2469

example:
```java
SeaTunnelRowType rowType = SeatunnelSchema.buildWithConfig(config).getSeaTunnelRowType();
```

Reply:
`SeatunnelSchema` does not support your notation yet;

Reply:
example：
https://github.com/apache/incubator-seatunnel/blob/dev/seatunnel-connectors-v2/connector-iotdb/src/main/java/org/apache/seatunnel/connectors/seatunnel/iotdb/source/IoTDBSource.java#L70

Reply:
You can merge from dev again. Then you can use this feature.